src/sbd-inquisitor.c

```diff
@@ -39,6 +39,40 @@ bool sync_resource_startup = false;
 
 int parse_device_line(const char *line);
 
+#define CHECK_RC(rc, errmsg, args...)       \
+    if (rc != 0) {                          \
+        fprintf(stderr, errmsg, ##args);    \
+        exit_status = -1;                   \
```

Comment:
in other cases where we detect that there is something fishy about cmdline options we exit -2 - right.
maybe time to define some nice macros for exit values ... but of course this can/should be an extra PR.